website/pages/docs/theming/scale-out-theme.mdx

```diff
@@ -0,0 +1,519 @@
+    400: "rgba(252, 157, 157, 0.24)",
+    500: "rgba(252, 157, 157, 0.36)",
+    600: "rgba(252, 157, 157, 0.48)",
+    700: "rgba(252, 157, 157, 0.64)",
+    800: "rgba(252, 157, 157, 0.80)",
+    900: "rgba(252, 157, 157, 0.92)",
+  },
+  // More...
+}
+
+export default colors
+```
+
+### 🌈 Border Radius
+You can customize border radius like this:
+
+```ts live=false
+// radius.ts
+
+const radii = {
+  none: "0",
+  base: "7px",
+  "circle": "50%",
+  // More...
+}
+
+export type Radii = typeof radii
+
+export default radii
+```
+
+### 🌓 Shadows
+You can add additional box-shadows like this:
+
+```ts live=false
+// shadows.ts
+
+const shadows = {
+  shopify: 'rgba(0, 0, 0, 0.15) 0px 5px 15px 0px',
+  airbnb: 'rgba(0, 0, 0, 0.15) 0px 2px 8px',
+  mac: 'rgba(0, 0, 0, 0.2) 0px 20px 30px',
+  // More...
+};
+
+export type Shadows = typeof shadows
+
+export default shadows
+```
+
+### 🚀 Spacing
+Setting the size of the whitespace for `padding`, `margin` and `gap` etc.
+
+```ts live=false
+// spacing.ts
+
+export const spacing = {
+  full: '100%',
+  half: '50%',
+  x1: '1em',
+  x2: '2em',
+  x4: '4em',
+  x8: '8em',
+  x16: '16em',
+  // More...
+}
+
+export type Spacing = typeof spacing
+```
+
+### 📏 Sizes
+You can customize the global sizing of components you build for your project.
+These value can be used by the `width`, `height`, and `maxWidth`, `minWidth`, `maxHeight`, `minHeight` styles.
+
+```ts live=false
+// sizes.ts
+import { spacing } from "./spacing"
+
+const largeSizes = {
+  full: "100%",
+  half: "50%",
+  "3xs": "14rem",
+  "2xs": "16rem",
+  xs: "20rem",
+  sm: "24rem",
+  md: "28rem",
+  lg: "32rem",
+  xl: "36rem",
+  "2xl": "42rem",
+  "3xl": "48rem",
+  // More...
+}
+
+const container = {
+  sm: "640px",
+  md: "768px",
+  lg: "1024px",
+  xl: "1280px",
+}
+
+const sizes = {
+  ...spacing,
+  ...largeSizes,
+  container,
+}
+
+export type Sizes = typeof spacing &
+  typeof largeSizes & { container: typeof container }
+
+export default sizes
+```
+
+### 📝 Typography
+To customize Typography options, the theme object supports the following keys:
+
+- `fonts` (font families)
+- `fontSizes`
+- `letterSpacings`
+- `lineHeights`
+- `fontWeights`
+
+```ts live=false
+const typography = {
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "Georgia, serif",
+    mono: "Menlo, monospace",
+  },
+
+  fontSizes: {
+    xs: "0.75rem",
+    sm: "0.875rem",
+    md: "1rem",
+    lg: "1.125rem",
+    xl: "1.25rem",
+    "2xl": "1.5rem",
+    // More...
+  },
+
+  letterSpacings: {
+    tighter: "-0.06em",
+    tight: "-0.03em",
+    normal: "0",
+    wide: "0.03em",
+    wider: "0.06em",
+    // More...
+  },
+
+  lineHeights: {
+    shorter: 1.3,
+    short: 1.5,
+    base: 1.7,
+    tall: 1.9,
+    taller: "2",
+    // More...
+  },
+
+  fontWeights: {
+    // Custom in special cases. Like this:
+    // normal: 400,
+    // bold: 700,
+  },
+
+}
+
+export type Typography = typeof typography
```

Comment:
Please reduce the amount of examples in here and remove the Typography type export.